src/hedera.c

```diff
@@ -4,33 +4,26 @@
 #include <os.h>
 #include <string.h>
 
-#include "lib_standard_app/crypto_helpers.h"
-
 #include "globals.h"
+#include "lib_standard_app/crypto_helpers.h"
```

Comment:
Better to add this in the `Makefile`:
`INCLUDES_PATH += ${BOLOS_SDK}/lib_standard_app`
Then you can simplify by
`#include "crypto_helpers.h"`

Reply:
Done.